src/pages/temporal-technologies-inc-security.js

```diff
@@ -71,6 +71,12 @@ export default function Security() {
                 </li>
               </ul>
             </div>
+            <div className={clsx("col col--5", styles.securityPageHeaders)}>
+              <h2>Advisories</h2>
+            </div>
+            <div className={clsx("col col--4", styles.justifyLeft)}>
+              There are no current security advisories.
```

Comment:
Suggest changing "There are" to "We have". ("There are" has fuzzy scope.)

Reply:
This will be getting an update in a couple of days once we _do_ have an advisory. I'll change the wording then.